code/game/objects/items/devices/megaphone.dm

```diff
@@ -58,100 +58,8 @@
 	return
 
 
-
-
 /*
-	SOUND SYNTH BELOW THIS LINE
+	SOUND SYNTH NO LONGER LIVES HERE
 ----------------------------------------------
+	HE NOW LIVES IN code/modules/assembly/sound_synth.dm
```

Comment:
Eh should probably just kill this comment whole.

megaphone.dm wasn't ever a good file for it anyways.

Not a place I'd look by far.